app/views/studies/index.html.erb

```diff
@@ -1,3 +1,8 @@
+<div id="my-studies-content"></div>
+<script type="text/javascript" nonce="<%= content_security_policy_script_nonce %>">
+  window.SCP.renderComponent('my-studies-content', 'MyStudiesPage')
+</script>
+<!--
```

Comment:
Blocker: despite this being commented out, the code is still executing.  You can prove this by going into `app/controllers/studies_controller.rb#L39` and commenting out line `@studies = Study.accessible(current_user).to_a`.  This will then throw an error on page reload.

Furthermore, that query is still happening in the background, causing the slowdown to still occur in loading all studies (especially for admins).  I think this new view is great, and there's no reason to keep around the legacy code.  It can still be found on older commits if we really need it, which I doubt.

Reply:
Great catch!